### DISCUSSION TITLE
How do I use persistent AWS S3 storage when using EC2 Cluster?

### DISCUSSION THREAD

**Discussion by elspeth-strathern** [milvus-io/milvus#31416] [Q&A and General discussion]
Or any persistent storage, if it's easier. I assume it's something to do with setting up nodes to use S3 endpoint instead of a minio node, but I can't see where to set those configuration details.

Any advice would be appreciated.

**Comment by LoveEachDay** [comment]
@james-strathern-omnieye Just replace the `minio` section configuration under `milvus.yaml` with corresponding s3 settings. Take some references from https://milvus.io/docs/deploy_s3.md#Configure-S3-with-Docker-Compose
